src/dev/build/tasks/bin/scripts/opensearch-dashboards

```diff
@@ -15,8 +15,15 @@ done
 
 DIR="$(dirname "${SCRIPT}")/.."
 CONFIG_DIR=${OSD_PATH_CONF:-"$DIR/config"}
-NODE="${DIR}/node/bin/node"
+
+if [ -x "${DIR}/node/bin/node" ]; then
+  NODE="/usr/local/bin/node"
+elif [ -x "$(which node)" ]; then
+  NODE="$(which node)"
+fi
+
 test -x "$NODE"
```

Comment:
Useless test.  The same test happen 2 lines bellow in a condition.  This linne can be removed.